src/frontend/apps/standalone_site/src/routes/routes.tsx

```diff
@@ -78,64 +70,154 @@ enum ERouteNames {
   MyOrganizations = 'MyOrganizations',
   MyContents = 'MyContents',
 }
+enum EMyContentsSubRouteNames {
+  Videos = 'Videos',
+  Lives = 'Lives',
+  VirtualClasses = 'VirtualClasses',
+  Lessons = 'Lessons',
+}
+
+export interface BasicRoute {
+  label: React.ReactNode;
+  path: string;
+  alias?: string[];
+  menuIcon?: React.ReactNode;
+}
+export interface Route extends BasicRoute {
+  subRoutes?: { [key in string]: BasicRoute };
+}
 
 type Routes = {
-  [key in ERouteNames]: Route;
+  [key in ERouteNames as Exclude<ERouteNames, 'MyContents'>]: BasicRoute;
+} & {
+  [ERouteNames.MyContents]: BasicRoute & {
+    subRoutes: {
+      [key in EMyContentsSubRouteNames]: BasicRoute;
+    };
+  };
```

Comment:
Why don't you type BasicRoute with an optional subRoutes ?

Reply:
The menu for the moment has only one deepness, I didnt want that "subroutes" get the possibilities to have another deepness (subroutes).
You get as well something more accurate so less error prone about what you can do with each route:
[routes.MyContents] has the property subRoutes:
 https://github.com/openfun/marsha/blob/79c561f1a824132ddae3421d20fbcd4a10bfdac7/src/frontend/apps/standalone_site/src/routes/AppRoutes.tsx#L45

[routes.Homepage] doesn't have the property subRoutes:
 https://github.com/openfun/marsha/blob/79c561f1a824132ddae3421d20fbcd4a10bfdac7/src/frontend/apps/standalone_site/src/routes/AppRoutes.tsx#L35

Reply:
If you want to stick to that, declare a subtype for the Omit to remove the cast